packages/main/src/DatePicker.ts

```diff
@@ -627,7 +617,7 @@ class DatePicker extends DateComponentBase {
 			return true;
 		}
```

Comment:
all public methods are not shown in the documentation.

Reply:
True, I guess with need to add "@name" or something to the public methods as well, because they are also extracted our of the class and without anything in addition the JSDoc plugin can't relate them to a method of a class.

Reply:
I’ve tried it with adding @name to all public methods but it doesn’t work. We need to discuss how to handle it.

Reply:
This works
```js
/**
	 * Closes the picker.
	 * @public
	 * @name sap.ui.webc.main.DatePicker#closePicker
	 * @function
	 */
	closePicker() {
		this.responsivePopover.close();
	}
```